### ISSUE TITLE
I will restart from scratch, using 20.04 ARM image on my VM , and work stop by step and record what issues I encounter.

### ISSUE THREAD

**Issue by TOAOQ** [rconfig/rconfig#2]
I will restart from scratch, using 20.04 ARM image on my VM , and work stop by step and record what issues I encounter.

_Originally posted by @TOAOQ in https://github.com/rconfig/rconfig/issues/1#issuecomment-1473633335_

**Comment by TOAOQ** [comment]
Installed fresh and clean VM from scratch using Ubuntu 20.04.5 LTS (GNU/Linux 5.4.0-144-generic aarch64)
Followed steps, again encountered issues on step 5:

Edit the .env file and update the following variables:
APP_URL="https://SERVER.DOMAIN.LOCAL" 
APP_DIR_PATH=/var/www/html/rconfig
DB_HOST=DBHOST 
DB_PORT=DBPORT
DB_DATABASE=DBNAME
DB_USERNAME=DBUSER
DB_PASSWORD=DBPASS

Servername was easy part, the rest I edited:
DB_HOST=localhost
DB_PORT=3306
DB_DATABASE=DBNAME
DB_USERNAME=root
DB_PASSWORD=my ownpassword for rootdb

Maybe I missed somehow what to record in an earlier step, and re-create here? Please explain

As there was no database configured on the server, I created a new DB called DBNAME just to satisfy that line.

After that the final step again gave me the same issues:
```

                                                                                                                       
                                                  APPLICATION INSTALL.                                                  
                                                                                                                        

   INFO  Discovering packages.  

  enlightn/enlightn ............................................................................................. DONE
  graham-campbell/github ........................................................................................ DONE
  laravel/horizon ............................................................................................... DONE
  laravel/passport .............................................................................................. DONE
  laravel/sanctum ............................................................................................... DONE
  laravel/tinker ................................................................................................ DONE
  laravel/ui .................................................................................................... DONE
  nesbot/carbon ................................................................................................. DONE
  nunomaduro/laravel-desktop-notifier ........................................................................... DONE
  nunomaduro/termwind ........................................................................................... DONE
  qruto/laravel-flora ........................................................................................... DONE
  spatie/laravel-activitylog .................................................................................... DONE
  spatie/laravel-health ......................................................................................... DONE
  spatie/laravel-ignition ....................................................................................... DONE


   INFO  Running actions.  

  command key:generate Set the application key .............................................................. 1ms DONE
  command migrate Run the database migrations 
............................................................. 206ms DONE
  command passport:install Run the commands necessary to prepare Passport for use ......................... 181ms DONE
  exec    if [ -f /etc/supervisord.d/horizon_supervisor.ini ]; then unlink /etc/supervisord.d/horizon_supervisor.ini; fi  2ms DONE
  exec    sed -i -e s+PWD+$PWD+g $PWD/horizon_supervisor.ini ................................................ 1ms DONE
  exec    sudo ln -s $PWD/horizon_supervisor.ini /etc/supervisord.d/horizon_supervisor.ini .................. 7ms FAIL
  exec    systemctl restart supervisord ..................................................................... 4ms FAIL
  exec    sed -i -e s+PWD+$PWD+g $PWD/rconfig-vhost.conf .................................................... 1ms DONE
  exec    if [ -f /etc/httpd/conf.d/rconfig-vhost.conf ]; then unlink /etc/httpd/conf.d/rconfig-vhost.conf; fi  1ms DONE
  exec    sudo ln -s $PWD/rconfig-vhost.conf /etc/httpd/conf.d/rconfig-vhost.conf ........................... 3ms FAIL
  exec    systemctl restart httpd ........................................................................... 4ms FAIL
  exec    chown -R apache:apache $PWD ....................................................................... 1ms FAIL
  command rconfig:clear-all Clear all cache and config elements for Laravel, NPM and other dependencies 
No config updates to processes
Failed to restart supervisord.service: Unit supervisord.service not found.

> post-autoload-dump: Illuminate\Foundation\ComposerScripts::postAutoloadDump
Generated optimized autoload files containing 6914 classes
"chown(): Unable to find uid for apache" // app/Http/Helpers.php:83
. 1,124ms DONE
  command rconfig:sync-tasks Sync the schedule of the app with the schedule monitor ......................... 1ms DONE
  script  cache ......................................................................................................
  command route:cache Create a route cache file for faster route registration .............................. 11ms DONE
  command config:cache Create a cache file for faster configuration loading ................................. 7ms DONE
  command event:cache Discover and cache the application's events and listeners ............................. 1ms DONE
  ....................................................................................................................
  script  build ......................................................................................................
  exec    npm install ....................................................................................... 1ms FAIL
  exec    npm run build ..................................................................................... 1ms FAIL
  ....................................................................................................................

  Assets publishing laravel-assets ...................................................................................  
  Copying directory [vendor/laravel/horizon/public] to [public/vendor/horizon] .................................. DONE


  Show errors? (yes/no) [no]
❯ y

  exec sudo ln -s $PWD/horizon_supervisor.ini /etc/supervisord.d/horizon_supervisor.ini ......................... FAIL  

In Process.php line 40:
                                                                                                             
  [RuntimeException (1)]                                                                                     
  ln: failed to create symbolic link '/etc/supervisord.d/horizon_supervisor.ini': No such file or directory  
                                                                                                             

Exception trace:
  at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Process.php:40
 Qruto\Flora\Actions\Process->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Action.php:40
 Qruto\Flora\Actions\Action->Qruto\Flora\Actions\{closure}() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/View/Components/Task.php:36
 Illuminate\Console\View\Components\Task->render() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/View/Components/Factory.php:58
 Illuminate\Console\View\Components\Factory->__call() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Action.php:58
 Qruto\Flora\Actions\Action->__invoke() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:156
 Qruto\Flora\RunInternal->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:72
 Qruto\Flora\RunInternal->start() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Console/Commands/FloraCommand.php:230
 Qruto\Flora\Console\Commands\FloraCommand->perform() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Console/Commands/FloraCommand.php:64
 Qruto\Flora\Console\Commands\FloraCommand->handle() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:36
 Illuminate\Container\BoundMethod::Illuminate\Container\{closure}() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/Util.php:41
 Illuminate\Container\Util::unwrapIfClosure() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:93
 Illuminate\Container\BoundMethod::callBoundMethod() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:37
 Illuminate\Container\BoundMethod::call() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/Container.php:661
 Illuminate\Container\Container->call() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Command.php:194
 Illuminate\Console\Command->execute() at /var/www/html/rconfig/vendor/symfony/console/Command/Command.php:312
 Symfony\Component\Console\Command\Command->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Command.php:164
 Illuminate\Console\Command->run() at /var/www/html/rconfig/vendor/symfony/console/Application.php:1022
 Symfony\Component\Console\Application->doRunCommand() at /var/www/html/rconfig/vendor/symfony/console/Application.php:314
 Symfony\Component\Console\Application->doRun() at /var/www/html/rconfig/vendor/symfony/console/Application.php:168
 Symfony\Component\Console\Application->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Application.php:102
 Illuminate\Console\Application->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php:155
 Illuminate\Foundation\Console\Kernel->handle() at /var/www/html/rconfig/artisan:37



  exec systemctl restart supervisord ............................................................................ FAIL  

In Process.php line 40:
                                                                              
  [RuntimeException (5)]                                                      
  Failed to restart supervisord.service: Unit supervisord.service not found.  
                                                                              

Exception trace:
  at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Process.php:40
 Qruto\Flora\Actions\Process->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Action.php:40
 Qruto\Flora\Actions\Action->Qruto\Flora\Actions\{closure}() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/View/Components/Task.php:36
 Illuminate\Console\View\Components\Task->render() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/View/Components/Factory.php:58
 Illuminate\Console\View\Components\Factory->__call() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Action.php:58
 Qruto\Flora\Actions\Action->__invoke() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:156
 Qruto\Flora\RunInternal->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:72
 Qruto\Flora\RunInternal->start() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Console/Commands/FloraCommand.php:230
 Qruto\Flora\Console\Commands\FloraCommand->perform() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Console/Commands/FloraCommand.php:64
 Qruto\Flora\Console\Commands\FloraCommand->handle() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:36
 Illuminate\Container\BoundMethod::Illuminate\Container\{closure}() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/Util.php:41
 Illuminate\Container\Util::unwrapIfClosure() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:93
 Illuminate\Container\BoundMethod::callBoundMethod() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:37
 Illuminate\Container\BoundMethod::call() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/Container.php:661
 Illuminate\Container\Container->call() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Command.php:194
 Illuminate\Console\Command->execute() at /var/www/html/rconfig/vendor/symfony/console/Command/Command.php:312
 Symfony\Component\Console\Command\Command->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Command.php:164
 Illuminate\Console\Command->run() at /var/www/html/rconfig/vendor/symfony/console/Application.php:1022
 Symfony\Component\Console\Application->doRunCommand() at /var/www/html/rconfig/vendor/symfony/console/Application.php:314
 Symfony\Component\Console\Application->doRun() at /var/www/html/rconfig/vendor/symfony/console/Application.php:168
 Symfony\Component\Console\Application->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Application.php:102
 Illuminate\Console\Application->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php:155
 Illuminate\Foundation\Console\Kernel->handle() at /var/www/html/rconfig/artisan:37



  exec sudo ln -s $PWD/rconfig-vhost.conf /etc/httpd/conf.d/rconfig-vhost.conf .................................. FAIL  

In Process.php line 40:
                                                                                                        
  [RuntimeException (1)]                                                                                
  ln: failed to create symbolic link '/etc/httpd/conf.d/rconfig-vhost.conf': No such file or directory  
                                                                                                        

Exception trace:
  at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Process.php:40
 Qruto\Flora\Actions\Process->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Action.php:40
 Qruto\Flora\Actions\Action->Qruto\Flora\Actions\{closure}() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/View/Components/Task.php:36
 Illuminate\Console\View\Components\Task->render() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/View/Components/Factory.php:58
 Illuminate\Console\View\Components\Factory->__call() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Action.php:58
 Qruto\Flora\Actions\Action->__invoke() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:156
 Qruto\Flora\RunInternal->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:72
 Qruto\Flora\RunInternal->start() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Console/Commands/FloraCommand.php:230
 Qruto\Flora\Console\Commands\FloraCommand->perform() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Console/Commands/FloraCommand.php:64
 Qruto\Flora\Console\Commands\FloraCommand->handle() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:36
 Illuminate\Container\BoundMethod::Illuminate\Container\{closure}() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/Util.php:41
 Illuminate\Container\Util::unwrapIfClosure() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:93
 Illuminate\Container\BoundMethod::callBoundMethod() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:37
 Illuminate\Container\BoundMethod::call() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/Container.php:661
 Illuminate\Container\Container->call() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Command.php:194
 Illuminate\Console\Command->execute() at /var/www/html/rconfig/vendor/symfony/console/Command/Command.php:312
 Symfony\Component\Console\Command\Command->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Command.php:164
 Illuminate\Console\Command->run() at /var/www/html/rconfig/vendor/symfony/console/Application.php:1022
 Symfony\Component\Console\Application->doRunCommand() at /var/www/html/rconfig/vendor/symfony/console/Application.php:314
 Symfony\Component\Console\Application->doRun() at /var/www/html/rconfig/vendor/symfony/console/Application.php:168
 Symfony\Component\Console\Application->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Application.php:102
 Illuminate\Console\Application->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php:155
 Illuminate\Foundation\Console\Kernel->handle() at /var/www/html/rconfig/artisan:37



  exec systemctl restart httpd .................................................................................. FAIL  

In Process.php line 40:
                                                                  
  [RuntimeException (5)]                                          
  Failed to restart httpd.service: Unit httpd.service not found.  
                                                                  

Exception trace:
  at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Process.php:40
 Qruto\Flora\Actions\Process->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Action.php:40
 Qruto\Flora\Actions\Action->Qruto\Flora\Actions\{closure}() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/View/Components/Task.php:36
 Illuminate\Console\View\Components\Task->render() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/View/Components/Factory.php:58
 Illuminate\Console\View\Components\Factory->__call() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Action.php:58
 Qruto\Flora\Actions\Action->__invoke() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:156
 Qruto\Flora\RunInternal->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:72
 Qruto\Flora\RunInternal->start() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Console/Commands/FloraCommand.php:230
 Qruto\Flora\Console\Commands\FloraCommand->perform() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Console/Commands/FloraCommand.php:64
 Qruto\Flora\Console\Commands\FloraCommand->handle() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:36
 Illuminate\Container\BoundMethod::Illuminate\Container\{closure}() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/Util.php:41
 Illuminate\Container\Util::unwrapIfClosure() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:93
 Illuminate\Container\BoundMethod::callBoundMethod() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:37
 Illuminate\Container\BoundMethod::call() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/Container.php:661
 Illuminate\Container\Container->call() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Command.php:194
 Illuminate\Console\Command->execute() at /var/www/html/rconfig/vendor/symfony/console/Command/Command.php:312
 Symfony\Component\Console\Command\Command->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Command.php:164
 Illuminate\Console\Command->run() at /var/www/html/rconfig/vendor/symfony/console/Application.php:1022
 Symfony\Component\Console\Application->doRunCommand() at /var/www/html/rconfig/vendor/symfony/console/Application.php:314
 Symfony\Component\Console\Application->doRun() at /var/www/html/rconfig/vendor/symfony/console/Application.php:168
 Symfony\Component\Console\Application->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Application.php:102
 Illuminate\Console\Application->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php:155
 Illuminate\Foundation\Console\Kernel->handle() at /var/www/html/rconfig/artisan:37



  exec chown -R apache:apache $PWD .............................................................................. FAIL  

In Process.php line 40:
                                        
  [RuntimeException (1)]                
  chown: invalid user: ‘apache:apache’  
                                        

Exception trace:
  at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Process.php:40
 Qruto\Flora\Actions\Process->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Action.php:40
 Qruto\Flora\Actions\Action->Qruto\Flora\Actions\{closure}() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/View/Components/Task.php:36
 Illuminate\Console\View\Components\Task->render() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/View/Components/Factory.php:58
 Illuminate\Console\View\Components\Factory->__call() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Action.php:58
 Qruto\Flora\Actions\Action->__invoke() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:156
 Qruto\Flora\RunInternal->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:72
 Qruto\Flora\RunInternal->start() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Console/Commands/FloraCommand.php:230
 Qruto\Flora\Console\Commands\FloraCommand->perform() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Console/Commands/FloraCommand.php:64
 Qruto\Flora\Console\Commands\FloraCommand->handle() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:36
 Illuminate\Container\BoundMethod::Illuminate\Container\{closure}() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/Util.php:41
 Illuminate\Container\Util::unwrapIfClosure() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:93
 Illuminate\Container\BoundMethod::callBoundMethod() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:37
 Illuminate\Container\BoundMethod::call() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/Container.php:661
 Illuminate\Container\Container->call() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Command.php:194
 Illuminate\Console\Command->execute() at /var/www/html/rconfig/vendor/symfony/console/Command/Command.php:312
 Symfony\Component\Console\Command\Command->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Command.php:164
 Illuminate\Console\Command->run() at /var/www/html/rconfig/vendor/symfony/console/Application.php:1022
 Symfony\Component\Console\Application->doRunCommand() at /var/www/html/rconfig/vendor/symfony/console/Application.php:314
 Symfony\Component\Console\Application->doRun() at /var/www/html/rconfig/vendor/symfony/console/Application.php:168
 Symfony\Component\Console\Application->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Application.php:102
 Illuminate\Console\Application->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php:155
 Illuminate\Foundation\Console\Kernel->handle() at /var/www/html/rconfig/artisan:37



  script build .................................................................................................. FAIL  

In Process.php line 40:
                            
  [RuntimeException (127)]  
  sh: 1: npm: not found     
                            

Exception trace:
  at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Process.php:40
 Qruto\Flora\Actions\Process->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Action.php:40
 Qruto\Flora\Actions\Action->Qruto\Flora\Actions\{closure}() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/View/Components/Task.php:36
 Illuminate\Console\View\Components\Task->render() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/View/Components/Factory.php:58
 Illuminate\Console\View\Components\Factory->__call() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Action.php:58
 Qruto\Flora\Actions\Action->__invoke() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:156
 Qruto\Flora\RunInternal->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:72
 Qruto\Flora\RunInternal->start() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Script.php:68
 Qruto\Flora\Actions\Script->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Script.php:41
 Qruto\Flora\Actions\Script->Qruto\Flora\Actions\{closure}() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Actions/Script.php:49
 Qruto\Flora\Actions\Script->__invoke() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:156
 Qruto\Flora\RunInternal->run() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/RunInternal.php:72
 Qruto\Flora\RunInternal->start() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Console/Commands/FloraCommand.php:230
 Qruto\Flora\Console\Commands\FloraCommand->perform() at /var/www/html/rconfig/vendor/qruto/laravel-flora/src/Console/Commands/FloraCommand.php:64
 Qruto\Flora\Console\Commands\FloraCommand->handle() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:36
 Illuminate\Container\BoundMethod::Illuminate\Container\{closure}() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/Util.php:41
 Illuminate\Container\Util::unwrapIfClosure() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:93
 Illuminate\Container\BoundMethod::callBoundMethod() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php:37
 Illuminate\Container\BoundMethod::call() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Container/Container.php:661
 Illuminate\Container\Container->call() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Command.php:194
 Illuminate\Console\Command->execute() at /var/www/html/rconfig/vendor/symfony/console/Command/Command.php:312
 Symfony\Component\Console\Command\Command->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Command.php:164
 Illuminate\Console\Command->run() at /var/www/html/rconfig/vendor/symfony/console/Application.php:1022
 Symfony\Component\Console\Application->doRunCommand() at /var/www/html/rconfig/vendor/symfony/console/Application.php:314
 Symfony\Component\Console\Application->doRun() at /var/www/html/rconfig/vendor/symfony/console/Application.php:168
 Symfony\Component\Console\Application->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Console/Application.php:102
 Illuminate\Console\Application->run() at /var/www/html/rconfig/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php:155
 Illuminate\Foundation\Console\Kernel->handle() at /var/www/html/rconfig/artisan:37


```

**Comment by TOAOQ** [comment]
Did a reboot, After loggin in as root I get presented with this info:

Welcome to Ubuntu 20.04.6 LTS (GNU/Linux 5.4.0-144-generic aarch64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

  System information as of Fri 17 Mar 2023 12:35:34 PM CET

  System load:             0.33
  Usage of /:              17.4% of 29.82GB
  Memory usage:            16%
  Swap usage:              0%
  Processes:               149
  Users logged in:         0
  IPv4 address for enp0s5: 10.211.55.12
  IPv6 address for enp0s5: fdb2:2c26:f4e4:0:21c:42ff:fe1e:d7d8

 * Strictly confined Kubernetes makes edge and IoT secure. Learn how MicroK8s
   just raised the bar for easy, resilient and secure K8s cluster deployment.

   https://ubuntu.com/engage/secure-kubernetes-at-the-edge

Expanded Security Maintenance for Applications is not enabled.

5 updates can be applied immediately.
To see these additional updates run: apt list --upgradable

4 additional security updates can be applied with ESM Apps.
Learn more about enabling ESM Apps service at https://ubuntu.com/esm

New release '22.04.2 LTS' available.
Run 'do-release-upgrade' to upgrade to it.


Last login: Fri Mar 17 12:16:11 2023


                   /@                   
                 @@@@@@@                
             %@@@@@@@@@@  @.            
         @@@@@@          ,@@@@@@        
     @@@@@@                   @@@@@@    
  @@@@(         *@@@@@@@          @@@@@ 
  @@@       @@@@@@@@.@@@@@@@%       @@@ 
  @@@     %@@@@@         @@@@@      @@@ 
  @@@     %@@@         @@@@@@@      @@@ 
  @@@     %@@@      @@@@@@@@@@      @@@ 
  @@@     %@@@      @@@@@@@@@@      @@@ 
   ,,      @@@@@@@, @@@@@@@@@@      @@@ 
 @@@@@@.       @@@@@@@@@@@       @@@@@@@
 @@@@@@@@          %@,           .@@@@@@
       &@@@@@#             %@@@@@       
           .@@@@@@     @@@@@@           
                @@@@@@@@@        

rConfig - Network Configuration Management
	


-- Ubuntu Version --
VERSION="20.04.6 LTS (Focal Fossa)"
VERSION_ID="20.04"
VERSION_CODENAME=focal

-- Apache Version --
Server version: Apache/2.4.41 (Ubuntu)
Server built:   2023-03-08T17:32:54

-- PHP Version --
PHP 8.1.17 (cli) (built: Mar 16 2023 14:38:17) (NTS)

-- MYSQL Version --
10.3.38-MariaDB

You are currently working in /var/www/html


total 24K
drwxr-xr-x  3 www-data www-data 4.0K Mar 17 12:33 .
drwxr-xr-x  3 www-data www-data 4.0K Mar 17 12:02 ..
-rwxr-xr-x  1 www-data www-data  11K Mar 17 12:02 index.html
drwxr-xr-x 13 www-data root     4.0K Mar 17 12:27 rconfig

**Comment by TOAOQ** [comment]
BTW, now retrying, but with Rocky Linux 8 x86 on VMware

**Comment by TOAOQ** [comment]
Installin on Rocky went bettter:
Still no real clue what to fill out and did the same:
DB_HOST=localhost
DB_PORT=3306
DB_DATABASE=DBNAME
DB_USERNAME=root
DB_PASSWORD=my ownpassword for rootdb

Then got an error the DB was not found so I created the DB manually and went on. Then got:

 APPLICATION INSTALL.


   INFO  Discovering packages.

  enlightn/enlightn ........................................................................................................................... DONE
  graham-campbell/github ...................................................................................................................... DONE
  laravel/horizon ............................................................................................................................. DONE
  laravel/passport ............................................................................................................................ DONE
  laravel/sanctum ............................................................................................................................. DONE
  laravel/tinker .............................................................................................................................. DONE
  laravel/ui .................................................................................................................................. DONE
  nesbot/carbon ............................................................................................................................... DONE
  nunomaduro/laravel-desktop-notifier ......................................................................................................... DONE
  nunomaduro/termwind ......................................................................................................................... DONE
  qruto/laravel-flora ......................................................................................................................... DONE
  spatie/laravel-activitylog .................................................................................................................. DONE
  spatie/laravel-health ....................................................................................................................... DONE
  spatie/laravel-ignition ..................................................................................................................... DONE


   INFO  Running actions.

  command key:generate Set the application key ............................................................................................ 2ms DONE
  command migrate Run the database migrations ............................................................................................. 3ms DONE
  command passport:install Run the commands necessary to prepare Passport for use ......................................................... 5ms DONE
  exec    if [ -f /etc/supervisord.d/horizon_supervisor.ini ]; then unlink /etc/supervisord.d/horizon_supervisor.ini; fi ................. 18ms DONE
  exec    sed -i -e s+PWD+$PWD+g $PWD/horizon_supervisor.ini ............................................................................. 10ms DONE
  exec    sudo ln -s $PWD/horizon_supervisor.ini /etc/supervisord.d/horizon_supervisor.ini ............................................... 28ms DONE
  exec    systemctl restart supervisord ............................................................................................... 1,312ms DONE
  exec    sed -i -e s+PWD+$PWD+g $PWD/rconfig-vhost.conf ................................................................................. 10ms DONE
  exec    if [ -f /etc/httpd/conf.d/rconfig-vhost.conf ]; then unlink /etc/httpd/conf.d/rconfig-vhost.conf; fi ............................ 8ms DONE
  exec    sudo ln -s $PWD/rconfig-vhost.conf /etc/httpd/conf.d/rconfig-vhost.conf ........................................................ 27ms DONE
  exec    systemctl restart httpd ..................................................................................................... 1,112ms DONE
  exec    chown -R apache:apache $PWD ................................................................................................... 187ms DONE
  command rconfig:clear-all Clear all cache and config elements for Laravel, NPM and other dependencies
No config updates to processes

> post-autoload-dump: Illuminate\Foundation\ComposerScripts::postAutoloadDump
Generated optimized autoload files containing 6914 classes
............................... 4,851ms DONE
  command rconfig:sync-tasks Sync the schedule of the app with the schedule monitor ....................................................... 2ms DONE
  script  cache ....................................................................................................................................
  command route:cache Create a route cache file for faster route registration ............................................................ 25ms DONE
  command config:cache Create a cache file for faster configuration loading ............................................................... 9ms DONE
  command event:cache Discover and cache the application's events and listeners ........................................................... 1ms DONE
  ..................................................................................................................................................
  script  build ....................................................................................................................................
  exec    npm install ..................................................................................................................... 8ms FAIL
  exec    npm run build ................................................................................................................... 8ms FAIL
  ..................................................................................................................................................

  No assets for publishing .........................................................................................................................


Installed NPM manually and re-ran the install script and finished the install without any hassle.
- Seems Ubuntu is not getting set-up correctly, or doesn't cope with the code
- DB needs to be set-up manually
- NPM needs to get installed before running `php artisan install`

**Comment by rconfig** [comment]
- [ x] Seems Ubuntu is not getting set-up correctly, or doesn't cope with the code
- [ x] DB needs to be set-up manually
- [ x] NPM needs to get installed before running php artisan install 
closed with pr #3
